straxen/misc.py

```diff
@@ -244,6 +244,26 @@ def _convert_to_datetime(time_widget, time_zone):
         return time, time_ns
 
 
+@strax.Context.add_method
+def extract_latest_comment(self):
+    """
+    Extract the latest comment in the runs-database. This just adds info to st.runs
+
+    Example:
+        st.extract_latest_comment()
+        st.select_runs(available=('raw_records'))
+    """
+    if self.runs is None or 'comments' not in self.runs.keys():
+        self.scan_runs(store_fields=('comments',))
+        latest_comments = _parse_to_last_comment(self.runs['comments'])
+        self.runs['comments'] = latest_comments
+    return self.runs
+
+
+def _parse_to_last_comment(comments):
+    return [(c[-1]['comment'] if hasattr(c, '__len__') else '') for c in comments]
```

Comment:
This guy is a bit complicated and take a sec to understand but I guess it is fine. I do not see any better solution.